UnleashClient/features/Feature.py

```diff
@@ -118,6 +118,8 @@ def get_variant(self, context: dict = None, skip_stats: bool = False) -> dict:
                 LOGGER.warning("Error selecting variant: %s", variant_exception)
         if not skip_stats:
             self._count_variant(cast(str, variant["name"]))
+
+        variant["is_feature_enabled"] = is_feature_enabled
```

Comment:
I don't think we need the mutation here, this is just a dict so a comprehension should work here and then we can remove the these two lines
```suggestion
        return {**variant, "is_feature_enabled": is_feature_enabled}
```

Reply:
Thanks, TIL.